spec/system/pghero_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'PGHero', :demo_only do
+  describe 'access' do
+    context 'with the appropriate environment variables set' do
+      before do
+        allow(ENV).to receive(:[]).with('PGHERO_USERNAME').and_return('link')
+        allow(ENV).to receive(:[]).with('PGHERO_PASSWORD').and_return('hyrule')
+      end
+
+      it 'fails without the basic auth credentials' do
+        visit pg_hero_path
+
+        expect(status_code).to eq 401
```

Comment:
Unfortunately it looks like this isn't testable because of how basic auth is implemented in PGHero: 

https://github.com/ankane/pghero/blob/6d1c688a0917a4a240c0599b5aecb9b4b8c0c15a/app/controllers/pg_hero/home_controller.rb#L7

Another option would be to enforce a password in all environments _except_ Development (which I think can be done through pghero's config), but I'm also ok just operating on trust and removing this test.